src/main/java/uk/gov/justice/hmpps/prison/api/model/RequestToUpdateAddress.java

```diff
@@ -0,0 +1,47 @@
+package uk.gov.justice.hmpps.prison.api.model;
+
+import io.swagger.annotations.ApiModelProperty;
+
+import java.time.LocalDate;
+
+public class RequestToUpdateAddress {
```

Comment:
one day when we have a consistent HMPPS API we can unify address descriptions and get rid of silly things like "flat"